ktor-server/ktor-server-core/src/io/ktor/sessions/SessionTransportCookie.kt

```diff
@@ -5,6 +5,11 @@ import io.ktor.http.*
 import java.time.*
 import java.time.temporal.*
 
+/**
+ * SessionTransport that will add a Set-Cookie header and will read Cookie header
```

Comment:
It better to avoid future tense in the docs. I, myself, often catch myself writing that way, but it really reads better in present, like this: "SessionTransport that adds a Set-Cookie header and reads Cookie header"